src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -53,6 +65,16 @@ public class ObjectModuleDetails
         /// </summary>
         public int WarningLevel => this.compilerCommandLine.WarningLevel;
 
+        /// <summary>
+        /// Returns whether optimizations were enabled as defined by the various /O switches in the compiland command line
```

Comment:
>command [](http://example.com/codeflow?start=110&length=7)

The doc comments in this file should all end with a period, have a moment to quickly fix that up?

Reply:
Sure.  Leave it better than you found it.